plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -222,6 +223,7 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 			).toBeVisible();
 		}
 
+		await page.waitForTimeout( 1000 ); // Wait for 1 second
 		// Save before going to the Variations tab to prevent variations from all attributes to be automatically created
 		await page.locator( '#save-post' ).click();
```

Comment:
I think you can just do...

```
await page.locator( '#save-post' ).click();
```

It should auto-wait. Again, not sure why the explicit timeout is needed. If it really is needed, a comment explaining why would be useful.